python/packages/ai/teams/ai/clients/llm_client.py

```diff
@@ -91,27 +90,19 @@ def __init__(self, options: LLMClientOptions) -> None:
 
         self._options = options
 
-    def add_function_result_to_history(self, memory: MemoryBase, name: str, results: Any) -> None:
+    def add_action_output_to_history(self, memory: MemoryBase, id: str, results: str) -> None:
```

Comment:
Not sure if we can delete `add_function_result_to_history` since it's public. @aacebo thoughts? I suspect it was meant as a helper function for customers and since it's public we can't delete it ad-hoc.

Reply:
I did a reference check and we're not using it anywhere internally but it is a fair concern. Happy to just add it as another function if thats the case

Reply:
it is a public function but the llmclient isn't a public class (we don't even reference it in our docs) so its fine to remove.